tox.ini

```diff
@@ -70,7 +70,7 @@ commands =
   nosetests -v --tests=test/visualiser
 
 [testenv:flake8]
-deps = flake8
+deps = flake8>=3.0.2
```

Comment:
This doesn't fix the issue going forward, as newer versions of flake8 can still break the build without code changes.